common/djangoapps/util/tests/test_db.py

```diff
@@ -209,6 +209,7 @@ def test_migrations_are_in_sync(self):
         make a migrationless release that'll require a separate migration
         release afterwards, this test doesn't fail.
         """
+        @unittest.skip("Migration will delete several models. Need to ship not referencing it first. LEARNER-2041")
```

Comment:
This skip has to be placed above the declaration of the method, right underneath where `override_settings` is used right now.

Reply:
Ups, my bad, not sure how I missed that ! Will change it right now. Thanks.